gtda/mapper/visualization.py

```diff
@@ -175,6 +179,10 @@ def plot_interactive_mapper_graph(pipeline, data, layout='kamada_kawai',
                                   plotly_kwargs=None):
     """Plotting function for interactive Mapper graphs.
 
+    Nodes are colored according to :attr:`color_variable`. By default, the
```

Comment:
Perhaps this is a good chance to add a single extra sentence explaining what the interactivity means or at least pointing out it involves the cover and clustering steps (only)?

Reply:
Good idea! I added a new sentence :)